src/Syntax.cpp

```diff
@@ -46,10 +46,33 @@ void Syntax::loadSyntaxRules(const YAML::Node &config)
         // Iterate through each rule in the category
         for (const auto &rule : rules)
         {
-            QString regex = QString::fromStdString(rule["regex"].as<std::string>());
-            QColor color(QString::fromStdString(rule["color"].as<std::string>()));
+
+            QString regex;
+            try{
```

Comment:
nit: In this project, I am using a specific format for coding style, which is the following:
```cpp
try
{
}
catch ()
{
}
```
```cpp
if ()
{
}
```